src/NzbDrone.Core/Download/Clients/qBittorrent/QBittorrentTorrent.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+
+namespace NzbDrone.Core.Download.Clients.QBittorrent
+{
+    public class QBittorrentTorrent
+    {
+        public String Hash { get; set; } // Torrent hash
```

Comment:
Things aren't currently strict, but we prefer, string, int, long and bool instead of String, Int32, Int64 and Boolean.